private-ai-services/README.md

```diff
@@ -0,0 +1,12 @@
+# Private AI Services
+
+Private AI Services is part of [VMware Private AI Foundation with NVIDIA 9.0](https://techdocs.broadcom.com/content/dam/broadcom/techdocs/us/en/pdf/vmware/private-ai/private-ai-nvidia/vmware-private-ai-foundation-with-nvidia-9-0.pdf).  More information is available in [this blog post](https://blogs.vmware.com/cloud-foundation/2025/06/19/private-ai-services-new-in-vmware-private-ai-foundation-with-nvidia-in-vcf-9-0/).
+
+Here we're publishing some example YAML files to help you get started with Private AI Services on VCF 9.
+
+You must have first installed and configured the Private AI Services service into a VCF Supervisor cluster.
+
+Then, within a tenant namespace, you can use:
+
+- [An example `PAISConfiguration`](paisconfiguration.yaml) to configure a tenant namespace
```

Comment:
`to configure a tenant namespace` -> `to configure Private AI Services in a tenant namespace`